themes/digital.gov/layouts/communities/single.html

```diff
@@ -33,139 +32,128 @@ <h1>{{ .Title | markdownify }} Community</h1>
+                  </svg> Connect with us
+                </h4>
+
+                {{- range $i, $e := .Params.community_list -}}
+                  <div class="join-community-form__body">
+
+                    <div class="join-community-form__body-eligibility">
+                      {{/*
+                        Checks to see if the community is public or govt only and sets the "terms" of the community, with an option to override the terms for each community by setting it in the front matter
+                      */}}
+
+                      {{/* PUBLIC */}}
+                      {{- if eq $e.type "public" -}}
+                        {{/* Checking to see if community terms are defined */}}
+                        {{- if $e.terms -}}
+                        <p>{{- $e.terms | markdownify -}}</p>
+                        {{- else -}}
+                        <p>Open to the public.</p>
+                        {{- end -}}
+                      {{- end -}}
 
-                    {{/* PUBLIC */}}
-                    {{- if eq $e.type "public" -}}
-                      {{/* Checking to see if community terms are defined */}}
-                      {{- if $e.terms -}}
-                      <p class="terms">{{- $e.terms | markdownify -}}</p>
-                      {{- else -}}
-                      <p class="terms">Open to the public.</p>
+                      {{/* GOVT */}}
+                      {{- if eq $e.type "government" -}}
+                        {{- if $e.terms -}}
+                        <p>{{- $e.terms | markdownify -}}</p>
+                        {{- else -}}
+                        <p>Open to anyone with a .gov or .mil email address.</p>
+                        {{- end -}}
                       {{- end -}}
-                    {{- end -}}
-
-                    {{/* GOVT */}}
-                    {{- if eq $e.type "government" -}}
-                      {{- if $e.terms -}}
-                      <p class="terms">{{- $e.terms | markdownify -}}</p>
-                      {{- else -}}
-                      <p class="terms">Open to anyone with a .gov or .mil email address.</p>
+
+                      {{/* GOVT_ONLY */}}
+                      {{- if eq $e.type "government_only" -}}
+                        {{- if $e.terms -}}
+                        <p>{{- $e.terms | markdownify -}}</p>
+                        {{- else -}}
+                        <p>Open to any government employee.</p>
+                        {{- end -}}
                       {{- end -}}
-                    {{- end -}}
-
-                    {{/* GOVT_ONLY */}}
-                    {{- if eq $e.type "government_only" -}}
-                      {{- if $e.terms -}}
-                      <p class="terms">{{- $e.terms | markdownify -}}</p>
-                      {{- else -}}
-                      <p class="terms">Open to any government employee.</p>
+                    </div>
+
+                    <div class="join-community-form__body-email">
```

Comment:
This is the footer or call to action for this component right? The class name might not necessarily reflect the intent.

Reply:
This class is now removed, not sure why it is there.